From d7c107aff35800cd571f69d56a118d9ec91bd152 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20Men=C5=A1=C3=ADk?= <pemensik@redhat.com>
Date: Fri, 19 Jul 2024 12:58:05 +0200
Subject: [PATCH] Remove unused openssl/engine.h header

---
 ldns-1.8.3/examples/ldns-nsec3-hash.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ldns-1.8.3/examples/ldns-nsec3-hash.c b/ldns-1.8.3/examples/ldns-nsec3-hash.c
index f9325d3..f04f1aa 100644
--- a/ldns-1.8.3/examples/ldns-nsec3-hash.c
+++ b/ldns-1.8.3/examples/ldns-nsec3-hash.c
@@ -18,7 +18,6 @@
 
 #ifdef HAVE_SSL
 #include <openssl/conf.h>
-#include <openssl/engine.h>
 #endif /* HAVE_SSL */
 
 #define MAX_FILENAME_LEN 250
-- 
2.45.2

